src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -425,39 +423,39 @@
       [{:type     :main
         :on-press #(re-frame/dispatch [:chat.ui/edit-message message])
         :label    (i18n/label :t/edit-message)
-        :icon     :main-icons/edit-context20
+        :icon     :main-icons2/edit-context20
```

Comment:
are you sure its icon2 ?we don't have `edit-context20`

Reply:
most likely old icons are used here, so it shouldn't be icons2 ns

Reply:
Both are working actually for this particular one. But, other icons are not working after the last icons commit, not sure why

Reply:
For both icons and icons2

Reply:
yes because we ignore namespace, its only for devs and IDE , so you should use icon2 namespace only when you use icon component from quo2, in that case `edit-context20` won't work because there is no such icon in new icons set